modules/admin_manual/pages/maintenance/manual_upgrade.adoc

```diff
@@ -225,28 +198,10 @@ visible at the bottom of your Admin page.
 * Re-enable your third-party apps.
 * Apply xref:installation/manual_installation.adoc#set-strong-directory-permissions[strong permissions] to your ownCloud directories.
 
-[[test-the-upgrade]]
-== Test the Upgrade
-
-Previous versions of ownCloud included a migration test. ownCloud first
-ran a migration simulation by copying the ownCloud database and
-performing the upgrade on the copy, to ensure that the migration would
-succeed.
-
-Then the copied tables were deleted after the upgrade was completed.
-This doubled the upgrade time, so admins could skip this test (by
-risking a failed upgrade) with `php occ upgrade --skip-migration-test`.
-
-NOTE: The migration test was removed in ownCloud 10.0. 
-
-ownCloud server
-admins should have current backups before migration, and rely on backups
-to correct any problems from the migration.
-
 [[reverse-upgrade]]
 == Reverse Upgrade
 
-If you need to reverse your upgrade, see restore.
+If you need to reverse your upgrade, see xref:maintenance/upgrading/manual_upgrade/restore.adoc[restore]
```

Comment:
The xref is wrong, use: `xref:maintenance/restore.adoc[Restoring owncloud] documenation.`